@xen-orchestra/lite/src/locales/en.json

```
@@ -28,9 +28,11 @@
   "stats": "Stats",
   "status": "Status",
   "storage": "Storage",
+  "storage-usage": "Storage usage",
   "switch-theme": "Switch theme",
   "system": "System",
   "tasks": "Tasks",
+  "top-5": "Top 5",
```

Maybe we could use interpolation?
```suggestion
  "top-#": "Top {n}",
```

Then in template:
```vue
<span>{{ $t("top-#", { n: 5 }) }}</span>
```